data/net/minecraft/world/item/BucketItem.mapping

```diff
@@ -1,4 +1,16 @@
 CLASS net/minecraft/world/item/BucketItem
+	METHOD <init> (Lnet/minecraft/world/level/material/Fluid;Lnet/minecraft/world/item/Item$Properties;)V
+		ARG 1 content
```

Comment:
so far every param was similar to the class tpye, maybe fluid instead of content?